functions/src/modules/pm/categories/prizes/listPrize.ts

```diff
@@ -0,0 +1,57 @@
+import type { ExpressFunction } from "../../../../@types";
+import { firebaseApp } from "../../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../../helpers";
+import { uasPermissionSwitch } from "../../../../systems/uas";
+
+const listPrize: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: {
+      accepted: execute,
+     
+    },
+  })(req, res, next);
+};
+
+const execute: ExpressFunction = (req, res, next) => {
+  const errorHandler = expressErrorHandlerFactory(req, res, next);
+
+  firebaseApp
+    .firestore()
+    .collection("Prize")
```

Comment:
This needs to be nested, so I think that it will end up as follows:
```suggestion
    .collection("prizeCategories)
    .doc(req.params.categoryId)
    .collection("prizes")
```